pkgs/development/libraries/xdg-desktop-portal-gtk/default.nix

```diff
@@ -35,6 +35,7 @@ stdenv.mkDerivation rec {
     gsettings-desktop-schemas
     gtk3
     gnome3.gnome-desktop
+    gnome3.gnome-settings-daemon
```

Comment:
Could you add a comment that it is for schemas for settings API?

Reply:
done.